ansible_runner/config/_base.py

```diff
@@ -505,7 +505,7 @@ def wrap_args_for_containerization(self, args, execution_mode, cmdline_args):
         self._update_volume_mount_paths(new_args, "{}".format(self.private_data_dir), dst_mount_path="/runner", labels=":Z")
 
         # Mount the stdout callback plugin from the ansible-runner code base
-        mount_paths = callback_mount(copy_if_needed=True)
+        mount_paths = callback_mount(copy_if_needed=True, copy_dir="{}/artifacts".format(self.private_data_dir))
```

Comment:
`$private_data_dir/artifacts` is a good candidate for copying the callback plugin to for the following reasons
1. clean up after run like tmpdir so no accumulation of garbage
2. have to be (and is therefore always) mountable by the container that are created